### ISSUE TITLE
BADREQUEST: Invalid fee per KB

### ISSUE THREAD

**Issue by ericrasor** [bitpay/copay#7617]
I cannot send money to a card.  Receiving the BADREQUEST: Invalid fee per KB when I try to send max.  I have a balance of 0.4575 that it will not allow me to use because of the error.

Prior to this problem, I sent 0.00577321 BTC (around $125 at that moment) at 15:31:52 on 12/21/2017.  The request was confirmed at 15:44 and all appeared okay.  When I opened the transaction up on my phone in BitPay it says 'This transaction amount is too small compared to current Bitcoin network fees.  Spending these funds will need a Bitcoin network fee cost comparable to the funds itself.'  So it appears the fees consumed the entire transfer because my total BTC went down.  

Please help me understand how and why this occurred.

UPDATE: I still cannot use 'send max'... same error (BADREQUEST: Invalid fee per KB), a little more information would have told me what the problem was and avoid most of my temper.

I was able to enter in a lesser amount and get it to process.  Problem is, the coins were worth 609.75 at the time I did the load and it would only allow me to load $500.  The fee I was charged was $51.45... (they say that is 8.86%, that does not work for my math).  Add the additional $20.90 that BitPay fee it makes it over 14%.  The fact that I had to TRY to load with no success, while the price plummeted, does not help with my temper.

With this kind of service the decentralized currency becomes a free-for-all on fees...  The wild west is alive and well and we are at the mercy of the thugs.  Ladies and gentlemen, if you don't want the government to jump in and take control, you better set some guidelines that are public, in-your-face obvious, transparent, reasonable and stick to them.

Show me how bad and how many will be raping me before you go to work.

**Comment by cleto123** [comment]
I just had the same situation with regards to the last example. I lost so much in fees and the deposit was never confirmed. So now I don't know where it is. It doesn't even show as valid on the blockchain, and the address doesn't show up either. This was on INSIGHT. I bought some bitcoin from a new bitcoin atm we got in town and sent it to my Bitpay wallet. It took a day, but was confirmed sent. As the process from the site in confirming everything, it disappeared. So now I'm going to email the National bitcoin atm company to see what happens now. I don't understand the Transaction Is To Small statement? Anyone

**Comment by rondonson** [comment]
Same is happening to me, this is really upsetting. I can see the amount in my wallet but when I try to add I get the same error. And the tx has over 70 confirmations, what gives?

**Comment by cleto123** [comment]
I just sent in another support ticket explaining everything

On Dec 22, 2017 12:11 PM, "rondonson" <notifications@github.com> wrote:

Same is happening to me, this is really upsetting. I can see the amount in
my wallet but when I try to add I get the same error. And the tx has over
70 confirmations, what gives?

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/bitpay/copay/issues/7617#issuecomment-353667474>, or mute
the thread

**Comment by Cr250cra1** [comment]
Why can't I spend my fuckin btc?

**Comment by got3nks** [comment]
Our wallet is stuck with the same error when sending transaction.
This urge a quick solution.

**Comment by sjrdmn** [comment]
I tried to spent some BTC with my Digital Bitbox. Same problem: "Invalid fee per KB".
Bitbox related to Copay? I also hope this is solved quickly.....

**Comment by erikbritto** [comment]
Please send me a message. I can guide you with some instructions.
My email eriksilva@aim.com

**Comment by got3nks** [comment]
Post it on here, why would anyone email you?

**Comment by erikbritto** [comment]
Because I solved mine.

**Comment by erikbritto** [comment]
Without risks.

**Comment by got3nks** [comment]
Let me rephrase the question: why would you not just post the solution here so that anyone can see and fix it?

**Comment by cryptorobb** [comment]
oh that easy @got3nks  it is so @erikbritto can scam you is my best guess, I am having the same issue first I attempted to make a payment, it said I didn't have sufficient btc (I had 0.01)  so I added a little more now I have 0.016 and now I get this BADREQUEST: Invalid fee per KB. Might have to stop using bitpay if it's going to give so much problems. and now with transaction fee being so expensive

**Comment by johnryan999** [comment]
I have am also having this problem.  I thought the GREAT thing about having this card was being able to instantly convert to dollars when needed.   If you google what the average bitcoin transaction cost is today it is about $33.  Bitpay wants to charge me right now $96.55 to load $250!!!  and if I choose the send the max amount option I get the error message of invalid fee per kb.

I would like to understand why Bitpay is charging triple the average fee.  I understand they need to make money but this is pure thuggery and unethical.  1) Is there any other more reasonable explanation than they are greedy with no foresight for customer retention? and 2) What are other fast options to convert bitcoin to dollars on a debit card?  Please help.

**Comment by cryptorobb** [comment]
![inkedphoto_2017-12-23_21-15-53_li](https://user-images.githubusercontent.com/34798907/34324417-ebfd69be-e826-11e7-83eb-b75ae8e2df27.jpg)
 Please tell me my eyes deceive me and it meant $5.39.  I am going to pay, and this will be the last time I ever use bitpay

**Comment by cryptorobb** [comment]
There is https://uquid.com/dash-debit-card which I want to get next, I just loaded 140 dollars with a 53 dollar transaction fee, This is the cause of the BADREQUEST its that you think that BitPay is reasonable and only gonna charge you some sane fee like 10 or 15 dollars to load your card. I loaded my bitpay visa anyway with the max amount I can, I am gonna shred this card after I withdraw all the money from it  and delete this app and never look back, you should all do the same.

**Comment by cholito1** [comment]
Use Electrum Wallet when sending btc to btc wallet. I just sent .00724906 btc to pay for a gift card. And put a fee of .0005 btc . I swear it had its first confirmation in 45secs or less. I really do believe the wallets we use make a difference. It should not but it really does work on electrum. Give it a try. I use bitpay too but lately not worth it. So instead. I send my btc from electrum with a real low free ex: .01btc w/ .00055 fee to Poloniex exchange to lite coin and withdraw to your Coinbase. Yes its a bit of work but you save your $50 bucks if not more     BTW report all this to the BBB many people have reported bitpay. Help us out and report. Don't let them run with your hard earn coins. Takes less then 10 mins

**Comment by erikbritto** [comment]
Hi all.

Being rude, unpolite or misbehavior don't make me rude or unpolite.
Sorry, I am not used to talk with scamers, or thieves or robbers.

First of all, stop using Copay, don't even open it.

Second, there needs some time for blockchain to "forgot" or return your bitcoins transactions. So be calm.
You sent a transfer "order" with a small fee, no one is accepting small fees. 
So wait the time to your "dead", "not serviced",  "not paying the minimum" or "not accepted" transfer orders to return.

Third, as soon as possible, install a Mycelium wallet. Then use the same twelve words that you used when creating the Copay wallet.  Start Myceliun wallet and do not create a new but restore on Mycelium wallet with the same twelve words /"seeds" that you used on Copay.

Fourth, wait the time to have your bitcoins back.

Fifth, resend the bitcoins with bigger fees when they are back on your Mycelium wallet. 

Just that!

**Comment by daughterovwater** [comment]
I am having the same issue!! Bitpay is a joke they have robbed me of so much money cuz of their hidden fees. I’m going to contact a lawyer and hopefully get a class-action lawsuit goin against them cuz this is crazy and can not be legal. Would anyone be interested in joining a class-action lawsuit against Bitpay?

**Comment by cholito1** [comment]
I’m doing the same, but before that report to the Better Business Bureau. The more we have complaining will help get us our money and prevent this again

**Comment by erikbritto** [comment]
Everything is okay for me and the buds that I helped.

**Comment by BGalliotti** [comment]
Hello, sorry for the delay,
We are closing this thread, please if you still need support, open a new one.
Thanks.
